client/src/components/exercise/lab8/pages/ExerciseStart.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { Component, Fragment } from "react";
+import { navigate } from "@reach/router";
+import { EXERCISE_PLAYING } from "../../../../constants/lab8";
+import { bindActionCreators } from "redux";
+import { actions as exerciseActions } from "../../../../reducers/lab8/ExerciseReducer";
+import { connect } from "react-redux";
+import PropTypes from "prop-types";
+
+class ExerciseStart extends Component {
```

Comment:
Small nitpick: the standard for the project is functional components, so this would look like:
`const ExerciseStart = (props) => { // component stuff }`
instead of the class declaration